transaction-api/src/main/java/com/example/auction/transaction/api/TransactionSummary.java

```diff
@@ -0,0 +1,31 @@
+package com.example.auction.transaction.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Value;
+
+import java.util.UUID;
+
+@Value
+public class TransactionSummary {
+
+    private final UUID userId;
+    private final UUID itemId;
+    private final UUID creatorId;
+    private final UUID winnerId;
+    private final String itemTitle;
```

Comment:
Isn't there one `UUID` too much?
It makes sense to have the request user, the creator and the winner in a command so that you can validate if the command can be fulfilled, but this is a `Summary` where the user making the request won't make much sense, will it?

Put another way: the creator and the winner should see the exact same `TransactionSummary` when requesting it.